.github/workflows/ci.yaml

```diff
@@ -40,26 +41,19 @@ jobs:
       - name: Post image to github packages
         run: |
           IMAGE_ID=ghcr.io/krawczowakris/$IMAGE_NAME
-          VERSION=latest
+          VERSION=${GITHUB_SHA::7}
           docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
           docker push $IMAGE_ID:$VERSION
 
-      - name: Send email about pipeline status
-        uses: dawidd6/action-send-mail@v3
-        with:
-          server_address: smtp.yandex.ru
-          server_port: 465
-          secure: true
-          username: ${{ secrets.MAIL_USERNAME }}
-          password: ${{ secrets.MAIL_PASSWORD }}
-          subject: Github Actions job result
-          to: ${{ secrets.EMAIL_ADDRESS }}
-          from: KrawczowaKris
-          body: Build job of ${{ github.repository }} completed with status ${{ job.status }}
+
+      - name: Getting an address to send an email
+        id: email
+        run: echo "email=$(sudo git log -n 1 --pretty='format:%ae')" >> $GITHUB_OUTPUT

```

Comment:
check env vars that github makes it available for every github action build. See official docs for github.

Reply:
@KrawczowaKris see if the following works for you
```
${{ github.event.pusher.email }}
```